frontend/src/components/SideNavBarButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+interface SideNavBarButtonProps {
+    svgIcon: React.ReactNode;
+    text: string;
+}
+export default function SideNavBarButton({svgIcon, text}: SideNavBarButtonProps) {
+  return (
+    <div className="side-nav-bar-button">
+      {svgIcon}
+      <div>{text}</div>
```

Comment:
This is good for now, but we'll have to change these to an actual button and then have a click event that will allow for navigation. We can wire all that up later though.